services/arrow-flight/src/test/java/com/dremio/service/flight/TestFlightSqlServerWithBasicAuth.java

```diff
@@ -21,14 +21,12 @@
 import org.apache.arrow.flight.FlightInfo;
 import org.apache.arrow.flight.sql.FlightSqlClient;
 import org.junit.BeforeClass;
-import org.junit.Ignore;
```

Comment:
Why are these changes made as part of this PR?

Reply:
In the previous PR we just refactored the producer without implementing any features, so these ignored test would fail when trying to make queries